src/mqt/qecc/cococo/snake_builder.py

```
@@ -0,0 +1,1472 @@
+        standard_steane_plaquettes = self.standard_steane()
+        for tile in self.positions:
+            z_stabilizers.extend(
+                {key: value for key, value in tile.items() if value in std_stab}
+                for std_stab in standard_steane_plaquettes
+            )
+
+        # add the weight-4, weight-2 stabilizers depending on the weight-8 X stabilizer's positions
+        compatible_weight_four = self.compatible_z_stabs_weight_four()
+        compatible_weight_two = self.compatible_z_stabs_weight_two()
+        x_stabs_weight_eight = [stab for stab in self.x_stabilizers if len(stab) == 8]
+        for i in range(len(self.positions) - 1):  # noqa: PLR1702
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `PLR1702`)

Remove unused `noqa` directive

(RUF100)